empiric-ui/src/pages/_app.tsx

```diff
@@ -46,13 +52,13 @@ const MyApp = ({ Component, pageProps }: AppProps) => {
       </Head>
       <DefaultSeo
         title="Empiric Network | ZK-Oracle"
-        description="Empiric Network is the first zk-native oracle, bringing the principles of DeFi to data infrastructure: decentralization, transparency and composability. Empiric is the leading oracle on Starknet, powered by high-quality data from the biggest market makers and exchanges, such as Alameda, CMT, Flow Traders, Gemini and Jane Street."
+        description="Empiric Network is the leading zk-native oracle live on StarkNet, bringing the principles of DeFi to data infrastructure: decentralization, transparency and composability."
         canonical="https://www.empiric.network"
         openGraph={{
           url: "https://www.empiric.network",
           title: "Empiric Network | ZK-Oracle",
           description:
-            "Empiric Network is the first zk-native oracle, bringing the principles of DeFi to data infrastructure: decentralization, transparency and composability. Empiric is the leading oracle on Starknet, powered by high-quality data from the biggest market makers and exchanges, such as Alameda, CMT, Flow Traders, Gemini and Jane Street.",
+            "Empiric Network is the leading zk-native oracle live on StarkNet, bringing the principles of DeFi to data infrastructure: decentralization, transparency and composability.",
```

Comment:
could add "price feed" in.

Empiric Network is the leading zk-native oracle, live on StarkNet with 20+ price feeds. Empiric makes data decentralized, transparent and composable.